### ISSUE TITLE
Host the addon outside of CurseForge

### ISSUE THREAD

**Issue by JVemmer** [Tercioo/World-Quest-Tracker#53]
As CurseForge is locking out everyone except Overwolf's own addon manager (source: [https://www.wowhead.com/news/ads-revenue-and-api-wowup-and-overwolf-split-over-addon-development-325912](https://www.wowhead.com/news/ads-revenue-and-api-wowup-and-overwolf-split-over-addon-development-325912)), please host this addon elsewhere than CurseForge (at least co-host it elsewhere). Options include:

- Hosting a .zip file of the addon here in the GitHub repository (this allows WowUp to pull it down)
- Hosting it on WowUpHub website
- Hosting it on wago.io (allows monetization to continue, if that is a concern)
- Hosting it on TukUI
- Hosting it on WowInterface

I am personally using WowUp to update addons, and while I very much respect that some addon creators want their work monetized, and believe that they should have the option, I do not support the Overwolf lock-out simply because they want everyone to use their app. If monetization is a priority, I suggest using wago.io as addon hosting, as it allows for monetization of addons, also while using WowUp; which shows they're willing to find solutions, rather than the Overwolf approach of refusing to cooperate.
I don't mind anyone hosting their addons on CurseForge, but please, please, please host it outside of CurseForge as well, for those of us who would like to cut ties with CurseForge, but still use your amazing addon :heart:

**Comment by Poisonenvy2** [comment]
Any update on this?  Great addon and would love to see it available to download somewhere other than from Curse.

**Comment by drewkeller** [comment]
I see it is on WowInterface, but the last update was in 2019. I am using WowUp which, for some reason, doesn't work with the zip files being used as releases here.

**Comment by Poisonenvy2** [comment]
>  Interestingly, Details! Damage Meter (also by Tercioo) DOES work in WowUp! Thumbs up for that :)

I don't think it is, yet, is it?  As far as I know, Details! is only available through Curse (other than a very old version on wowinterface)

**Comment by drewkeller** [comment]
https://github.com/Tercioo/Details-Damage-Meter..  I thought I had imported it from GitHub last night, but you're right, the one on GitHub is a couple years old and WowUp didn't like that zip file, either.

**Comment by JVemmer** [comment]
Since this issue seem have been jacked by details comments, I just want to say that Details is available on Wago :heart: Don't forget to download any plugins you need, as details on wago is standalone instead of the bundle coming from curse. Sadly don't see World Quest Tracker there yet, but I am still hoping!

**Comment by Poisonenvy2** [comment]
Yeah I use Details! from Wago now.  This addon (World Quest Tracker) doesn't appear to have been updated for a while, and there's probably no need to now, either.  Just have to hope that any similar addon created for DF gives us multiple download options.
